frontend/src/Models/AccountModel.tsx

```diff
@@ -1,5 +1,11 @@
 export interface IAccount {
-  id: number;
+  id: string;
   name: string;
   initialBalance: number;
 }
+
+export const createEmptyAccount = (): IAccount => ({
+  name: "",
+  initialBalance: NaN,
```

Comment:
Man atrodo mes dar neisprendem point of discussion ar nebutu geriau daryti `initialBalance` kaip atskira property ar kaip automatiskai idedama transaction. @Andrius-B ?

Reply:
tai galėsim balsavimo ceremoniją prasukt kai susitiksim, mano nuomonę jūs žinot :d

Reply:
Mano nuomone tbh pasikeite i tavo nuomone